### PR TITLE
Styled Components and improved code for comparison modal.

### PR DESCRIPTION
**Pull request by Acid-Override**

### CODE REVIEW

**Comment by addisonhernandez** [comment]
[Sat Apr 9 - Merge Party 🥳](https://trello.com/c/kUPtJK5x/44-sat-apr-9-merge-party-%F0%9F%A5%B3)

File: client/src/components/RelatedItems/Modal/ComparisonModal.jsx [hunk 1]
```diff
@@ -21,48 +24,32 @@ function Comparison({ setOpenModal, selectedComparisonItem }) {
...
         productInfo.features.map((item) => {
-          if (item.feature) {
-            if (item.value) {
-              return `${item.feature} : ${item.value}`;
-            }
-            return item.feature;
-          }
-          return item.value;
-        })
+          if (item.feature && item.value) return `${item.feature} : ${item.value}`;
```
Comment (positivefx) [inline client/src/components/RelatedItems/Modal/ComparisonModal.jsx:45]: noice clean lol

File: client/src/components/RelatedItems/StyledComponents/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  padding: 15px 32 px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  cursor: pointer;
+  transition: 0.5s all ease-out;
+`;
+
+export const StyledRightArrow = styled(FaArrowAltCircleRight)`
```
Comment (positivefx) [inline client/src/components/RelatedItems/StyledComponents/index.js:17]: ooo you can style icons like that

File: client/src/components/RelatedItems/RelatedProducts/ArrowNav.jsx [hunk 3]
```diff
@@ -1,16 +1,16 @@
 /* eslint-disable react/prop-types */
 import React from 'react';
-import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
-
+import { StyledRightArrow, StyledLeftArrow } from '../StyledComponents';
```
Comment (addisonhernandez) [inline client/src/components/RelatedItems/RelatedProducts/ArrowNav.jsx:3]: :fire:

File: client/src/components/RelatedItems/StyledComponents/index.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
+
+export const StyledButton = styled.button`
+  border: 2px solid ##d8cba7ff;
```
Comment (addisonhernandez) [inline client/src/components/RelatedItems/StyledComponents/index.js:5]: you have two # here!

**Review by addisonhernandez — APPROVED** [review]

**Pull request merged**